content/2_Bronze/Intro_Sets.mdx

```diff
@@ -244,6 +244,8 @@ In a C++ [`map`](http://www.cplusplus.com/reference/map/map/) `m`:
 - the `m[key] = value` operator assigns a value to a key and places the key and
   value pair into the map. The operator `m[key]` returns the value associated
   with the key. If the key is not present in the map, then `m[key]` is set to 0.
+  Please read [this article](https://stackoverflow.com/questions/10124679/what-happens-if-i-read-a-maps-value-where-the-key-does-not-exist)
+  to find out why `m[key]` is set to 0.
```

Comment:
I don't think it's accurate to refer to this as an "article?" Why not "question?"

Reply:
Can you fix it pls?